src/main/kotlin/com/exactpro/th2/service/generator/protoc/run/PluginRun.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.th2.service.generator.protoc.run
+
+import com.exactpro.th2.service.generator.protoc.Generator
+import com.exactpro.th2.service.generator.protoc.util.javaPackage
+import com.google.protobuf.compiler.PluginProtos
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+import java.util.Properties
+
+class PluginRun(private val inputStream: InputStream, private val outputStream: OutputStream, private val generators: List<Generator>) {
```

Comment:
I think we should pass `inputStream` and `outputStream` as parameters to the `generateResponse` method instead of passing them through the constructor